wdl2cwl/main.py

```diff
@@ -73,23 +73,26 @@ def load_wdl_workflow(self, obj: WDL.Tree.Workflow) -> cwl.Workflow:
         outputs: List[cwl.WorkflowOutputParameter] = []
         wf_steps: List[cwl.WorkflowStep] = []
         wf_name = obj.name
-        wf_description = obj.meta["description"]
+        wf_description = obj.meta["description"] if "description" in obj.meta else None
         for call in obj.body:
             call_name = call.name  # type: ignore
             callee = call.callee  # type: ignore
-            namespace, _ = call.callee_id  # type: ignore
+            namespace = ""
+            if len(call.callee_id) == 2:  # type: ignore
+                namespace, _ = call.callee_id  # type: ignore
+                namespace += "."
```

Comment:
This is getting messy. Maybe time to use https://github.com/common-workflow-lab/wdl-cwl-translator/issues/142 since miniWDL already calculates the entire list of workflow inputs?